tensorflow/python/data/kernel_tests/range_test.py

```diff
@@ -67,6 +68,13 @@ def testStopLessThanStartWithNegativeStep(self):
     dataset = dataset_ops.Dataset.range(start, stop, step)
     self.assertDatasetProduces(dataset, expected_output=range(10, 2, -1))
 
+  def testOutputType(self):
```

Comment:
my suggestion was to avoid adding a new test `testOutputType` and instead parametrize each of the above test with different output types. After my https://github.com/tensorflow/tensorflow/commit/2a2c812ab2330c9aac33335f10679a346436acfb, you can do this by replacing:

```
@combinations.generate(test_base.default_test_combinations())
def testStop(self):
  dataset = dataset_ops.Dataset.range(5)
  ...
```

with:

```
@combinations.generate(combinations.times(
    test_base.default_test_combinations(),
    combinations.combine(output_type=[dtypes.int32, dtypes.int64, dtypes.float32, dtypes.float64])))
def testStop(self, output_type):
  dataset = dataset_ops.Dataset.range(5, output_type=output_type)
  ...
```